Sources/SwifterSwift/UIKit/UIViewExtensions.swift

```diff
@@ -579,7 +578,52 @@ public extension UIView {
     func ancestorView<T: UIView>(withClass name: T.Type) -> T? {
         return ancestorView(where: { $0 is T }) as? T
     }
+}
+
+// MARK: - findConstraint and friends
+public extension UIView {
+    /// SwifterSwift: Search constraints until we find one for the given view and attribute. This will enumerate ancestors since constraints are always added to the common ancestor.
```

Comment:
Can you break this into two lines? :)

Reply:
Certainly! I was copying the style from elsewhere in the project, though. Another example where swiftformat would help.